terraform-cli/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM registry.access.redhat.com/ubi8/ubi
+
```

Comment:
so a couple of things : 

1) you may want to use ubi-minimal, unless you need more?

2) you don't need to install wget, curl is available (so curl -O would behave the same)

3) Make sure you do some yum update before i.e: https://github.com/tektoncd/catalog/blob/master/s2i/Dockerfile#L4 so you don't get show as security issue in quay.

4) Make sure you do yum clean all after any yum install so your images is slighty smaller,

5) it would be great if by default we are able to install latest stable, instead of having a static version number getting outdated quickly?

Sorry for nitpicking but those are good habits in general to take when writing Dockerfile,

other than that things looks fine, thanks 👍 

have the `yum install wget zip` before the `yum clean all`

(so it would not show as security issue in quay and make the image smaller)

Reply:
I appreciate all the remarks, will use this to improve my Dockerfile writing, just to point out two things

1. ubi-minimal doesn't have yum :sweat_smile: that's why I use this version
5. I couldn't find a link to a stable version on hashicorp's site, they only provide the binary in that naming format, I agree that this is bad for maintaining